Source/Tools/relive_api/relive_api.hpp

```diff
@@ -98,9 +98,16 @@ namespace ReliveAPI
     void DebugDumpTlvs(const std::string& prefix, const std::string& lvlFile, s32 pathId);
 
     API_EXPORT [[nodiscard]] s32 GetApiVersion();
+
+    API_EXPORT void ExportPathBinaryToJson(std::vector<u8>& fileDataBuffer, const std::string& jsonOutputFile, const std::string& inputLvlFile, s32 pathResourceId);
```

Comment:
if `std::vector<u8>& fileDataBuffer` was to speed up the tests then I think there needs to be an overload that doesn't have this argument as the things calling this API will now need updating and its less obvious what is happening

Reply:
There is, is on line 103 :)